remote-api/src/main/java/org/alfresco/rest/api/model/rules/RuleExecution.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * #%L
+ * Alfresco Remote API
+ * %%
+ * Copyright (C) 2005 - 2022 Alfresco Software Limited
+ * %%
+ * This file is part of the Alfresco software.
+ * If the software was purchased under a paid Alfresco license, the terms of
+ * the paid license agreement will prevail.  Otherwise, the software is
+ * provided under the following open source license terms:
+ *
+ * Alfresco is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Alfresco is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Alfresco. If not, see <http://www.gnu.org/licenses/>.
+ * #L%
+ */
+package org.alfresco.rest.api.model.rules;
+
+import java.util.Objects;
+
+import org.alfresco.service.Experimental;
+
+@Experimental
+public class RuleExecution
+{
+    private boolean eachSubFolderIncluded;
+    private boolean eachInheritedRuleExecuted;
```

Comment:
I thought that these were going to be `isEach...`?

Reply:
Setters and getters simulates that this names looks like `isEach...`.

Reply:
Ah - ok. So the setters and getters don't follow the Java convention. (Although I'm not sure having `isIs...` is a good getter name anyway :smile: ). I'm fine with this.

Reply:
Luckily IDE has generated `getIs...` 😄